inc/App/ExecutionTime.php

```diff
@@ -11,7 +11,11 @@ function __construct() {
 	public static function save_average_query_execution_time() {
 		global $wpdb;
 
-		$average = $wpdb->total_query_time / $wpdb->num_queries;
+		$average = null;
+
+		if( !empty( $wpdb->total_query_time ) && !empty( $wpdb->num_queries ) && $wpdb->num_queries > 0 ) {
+			$average = $wpdb->total_query_time / $wpdb->num_queries;
+		}
```

Comment:
I would argue that the divide by zero means that there have been no database queries in this run. Which means we shouldn't try to save (`NULL`, `0`) to the DB as it would not be useful for the graph / stats, we should just return if `num_queries` is `0` I think.

Open to discussion over it though.